app/ui/shared/widgets/overview/summaries/star.jsx

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import React, { Component, PropTypes } from 'react';
+import PureRenderMixin from 'react-addons-pure-render-mixin';
+
+import Btn from '../../../../shared/widgets/btn';
+
+const STAR_IMAGES = {
+  full: 'glyph-ratings-full.svg',
+  half: 'glyph-ratings-half.svg',
+  empty: 'glyph-ratings-empty.svg',
+};
+
+class Star extends Component {
+  constructor(props) {
+    super(props);
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate.bind(this);
```

Comment:
`Btn` is taking regular js objects as props, so shallow compares won't work because that would end up always allowing rerenders. For components that take in immutable.js objects or js primitives as props, PureRenderMixin is adequate.